maps/Behemoth.MX2.lua

```diff
@@ -0,0 +1,4 @@
+SetDaysAfterTownLost(GetPlayer(0), 30);

```

Comment:
What's the puprose of this? I am not sure when will this is called since this is not under any callback, but it is supposed to be under one of them